scripts/build-search-index.js

```diff
@@ -11,21 +11,25 @@ const SKIP_THESE = ["/menu", "/404", "/500"];
 
 function getAllFilesInDirectory(articleDirectory, files) {
   files = files ? files : [];
-  fs.readdirSync(articleDirectory).forEach(function (file) {
-    const subpath = path.join(articleDirectory, file);
-    if (fs.lstatSync(subpath).isDirectory()) {
-      getAllFilesInDirectory(subpath, files);
-    } else {
-      files.push(subpath);
-    }
-  });
-  return files;
+  try {
```

Comment:
I don't think this does anything. Remove?

Reply:
Removed on [8c6d551](https://github.com/streamlit/docs/pull/508/commits/8c6d551bf0316e3edf1d70c19e8c37b9e524d5a2)!